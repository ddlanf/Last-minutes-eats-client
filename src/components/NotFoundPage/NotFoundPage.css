.no-page-found{
    margin-top: 20%;
    text-align: center;
}