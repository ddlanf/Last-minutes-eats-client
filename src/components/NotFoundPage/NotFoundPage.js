import React, { Component } from 'react'
import './NotFoundPage.css'

export default class NotFoundPage extends Component {
    render() {
        return (
            <div>
                <h1 className="no-page-found">No page found</h1>
            </div>
        )
    }
}
