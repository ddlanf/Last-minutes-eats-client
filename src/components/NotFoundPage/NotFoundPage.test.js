import React from 'react';
import ReactDOM from 'react-dom';
import NotFoundPage from './NotFoundPage';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom'

describe('Make component', () => {
    

    it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<NotFoundPage/>, div);
    ReactDOM.unmountComponentAtNode(div);
    });

    it('renders the UI as expected', () => {
        const tree = renderer
          .create(<NotFoundPage/>)
          .toJSON();
        expect(tree).toMatchSnapshot();  
      });
});