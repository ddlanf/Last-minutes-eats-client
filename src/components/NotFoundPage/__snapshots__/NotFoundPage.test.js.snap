// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Make component renders the UI as expected 1`] = `
<div>
  <h1
    className="no-page-found"
  >
    No page found
  </h1>
</div>
`;
